src/LndClient/Data/Newtype.hs

```diff
@@ -231,10 +272,10 @@ instance ToGrpc (TxId a) ByteString where
   toGrpc = Right . coerce
 
 instance ToGrpc (Vout a) Word32 where
-  toGrpc = Right . coerce
+  toGrpc x = first (ToGrpcError . ("Vout overflow: " <>) . Universum.show) (tryFrom @Natural @Word32 $ unVout x)
```

Comment:
Nice! If it is possible, hide `coerce` how it's done with `fromIntegral` - coerce is safe to use, but we are moving from it into more clear accessors for newtypes